dist/report.d.ts

```diff
@@ -173,4 +173,12 @@ export declare class Report extends embed.Embed implements IReportNode, IFiltera
      * @returns {Promise<void>}
      */
     switchMode(viewMode: models.ViewMode): Promise<void>;
+    /**
+    * Refreshes data sources for the report.
```

Comment:
Let's change the comment as it doesn't trigger dataset refresh behind the scenes.